project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.11
+sbt.version=0.13.12
```

Comment:
I remember now, it was related to nice-sbt-settings at some point and is not relevant anymore.